client/control/constants.js

```diff
@@ -0,0 +1,10 @@
+export const BASE_API_URL = '/api/v1/';
+
+export const API_REQUEST_SETTINGS = {
+  credentials: 'include',
+  headers: {
+    'X-CSRFToken': document.getElementsByTagName('html')[0].dataset.csrf,
```

Comment:
This will be different every page request, so it can't be a constant. We might be able to move it into the `app-data` script tag, but this chunk is just copy/paste to move this to a re-usable place, so I didn't bother.